Add Chainguard OS Packages page plus reorg content (#3146)
chainguard-dev/edu: content/chainguard/chainguard-images/features/packages/private-apk-repos/index.md

Chainguard's private APK repositories are available to all Chainguard Containers customers.

## Contents of Private APK Repositories

> Your private APK repository contains all of the packages that are in the container images your organization is entitled to, plus some extra utilities, like `curl`.

Reviewer: ```suggestion
Your private APK repository contains all packages available in the container images your organization is entitled to, plus some extra utilities, like `curl`.
```
